dot4gravity/src/lib.rs

```diff
@@ -399,7 +425,7 @@ impl<Player: PartialEq + Clone> Game<Player> {
         side: Side,
```

Comment:
We can change `side` to take a `&Side` and also change `Self::can_drop_stone` to also take a `&Side` in order to avoid cloning the `side` variable.

Reply:
Nice catch! Thanks :pray: